lerobot/common/envs/utils.py

```diff
@@ -47,6 +47,8 @@ def preprocess_observation(observations: dict[str, np.ndarray]) -> dict[str, Ten
             # TODO(aliberts, rcadene): use transforms.ToTensor()?
             img = torch.from_numpy(img)
 
+            if img.dim() == 3:
+                img = img.unsqueeze(0)
```

Comment:
When preprocessing observations in a non-vectorized environment, especially in human-in-the-loop RL where there is only one environment, we need to explicitly add a batch dimension to the images.